### ISSUE TITLE
Render if DEBUG statement

### ISSUE THREAD

**Issue by javitosanchez** [dotnet/templating#432]
I've been reviewing the wiki docs, but didn't find an explanation. 
I'm trying to render:
```
#if DEBUG
            if (System.Diagnostics.Debugger.IsAttached)
            {
                _app.DebugSettings.EnableFrameRateCounter = true;
            }
#endif
```

But after generating the content is missing. I assume the engine is managing as a symbol. What is the way to get this working?

Thanks!!!

**Comment by mlorbetske** [comment]
Try this

```
//-:cnd
#if DEBUG
            if (System.Diagnostics.Debugger.IsAttached)
            {
                _app.DebugSettings.EnableFrameRateCounter = true;
            }
#endif
//+:cnd
```

**Comment by javitosanchez** [comment]
Works partially, I get the following generated code:

```
//-:cnd
//#if DEBUG
            if (System.Diagnostics.Debugger.IsAttached)
            {
                _app.DebugSettings.EnableFrameRateCounter = true;
            }
//#endif
//+:cnd
```

Is there a way to not get the `cnd` comment?

**Comment by mlorbetske** [comment]
There is a way, but in the currently released engine it wouldn't be pleasant (the default syntax for it was omitted and adding it requires redefining a lot of stuff correctly) - if you need that I can go over it. The easier way (if you're self hosting the engine) is to pick up the latest engine packages and use //-:cnd:noEmit instead. The syntax was added back by a recent PR.

**Comment by RehanSaeed** [comment]
@mlorbetske When is this going to get released. I want to port my main template but can't without this.

**Comment by mlorbetske** [comment]
@RehanSaeed this will accompany the next release of .NET Core - I'm not certain if I'm allowed to give a particular date at this time.

**Comment by mlorbetske** [comment]
Closing this as it was fixed by #433
